README.md

```diff
@@ -29,18 +29,29 @@
 
 ### Local Machine 
 
-- Set the application deployment (assuming a live K8s cluster w/ app) UUID as ENV (APP_UUID)
-
-- Set the ChaosEngine CR name as ENV (CHAOSENGINE) 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine.yaml
-
-- If the experiments are not executed, apply the ChaosResult CRs manually 
-  - For CR spec, see: https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosresult.yaml
-
-- Run the exporter container (litmuschaos/chaos-exporter:ci) on host network. It is necessary to mount the kubeconfig
-  & override entrypoint w/ `./exporter -kubeconfig <path>`
-
-- Execute `curl 127.0.0.1:8080/metrics` to view metrics
+- Fork this repo!
+- Edit the GOPATH variable and help it point to your workspace.
+- Clone the repository where you set the workspace.
+- Use the `git clobe repo-name` ,or just copy the git clone URL avaliable on the right side.
```

Comment:
Considering this is a README for "getting started", more than "contributing", can we summarize the 1st 4 points by saying something like: 

- Clone this repo into your $GOPATH/litmuschaos" 
  ``` git clone https://github.com/litmuschaos/chaos-exporter ``` 

(fork is something we would expect an aspiring contributor to do, and not all visitors/users of the repo may be one. However, good one to put into the contributing.md)